projects/plugins/boost/app/assets/src/js/features/page-cache/page-cache.tsx

```diff
@@ -97,17 +96,23 @@ const PageCache = () => {
 							'jetpack-boost'
 						) }
 					</p>
-					{ site.isAtomic && (
+					{ ( isWpCloudClient() || isWoaHosting() ) && (
 						<Notice
-							level="warning"
-							title={ __( 'Page Cache is unavailable', 'jetpack-boost' ) }
+							level={ isWoaHosting() ? 'success' : 'info' }
+							title={
+								isWoaHosting()
+									? __( 'Page Cache is running', 'jetpack-boost' )
+									: __( 'Page Cache is unavailable', 'jetpack-boost' )
+							}
 							hideCloseButton={ true }
 						>
 							<p>
-								{ __(
-									'Your website already has a page cache running on it powered by WordPress.com.',
-									'jetpack-boost'
-								) }
+								{ isWoaHosting()
+									? __(
+											'Your website already has a page cache running on it powered by WordPress.com.',
+											'jetpack-boost'
+									  )
+									: __( 'Your hosting provider already provides page caching.', 'jetpack-boost' ) }
 							</p>
 						</Notice>
```

Comment:
Optional: Have two separate notice components, one for WoA, one for WpCloud. This will avoid all those ternary operations and we will be able to see each notice by itself.

Reply:
Updated in bf4fda5.